### PR TITLE
Drilbert ii diggeredoo

### PR DESCRIPTION
**Pull request by tabreturn**
# New Port for Drilbert II Diggeredoo

## Game Information
- **Title**: Drilbert II Diggeredoo
- **URL**: https://wheybags.itch.io/drilbert-ii-diggeredoo

### CFW Tests
Ensure your game has been tested on all major CFWs:
- [x] AmberELEC
- [x] ArkOS
- [x] ROCKNIX
- [x] muOS

### Resolution Tests
Test all major resolutions:
- [ ] 480x320
- [x] 640x480
- [x] 720x720 (RGB30)
- [x] Higher resolutions (e.g., 1280x720)

## File Structure
- Your port should have the following structure:
  - portname/
    - port.json
    - README.md
    - screenshot.jpg
    - cover.jpg
    - gameinfo.xml
    - Port Name.sh
    - portname/
      - <portfiles here>

## Additional Resources
For an in-depth guide on creating a pull request, refer to: [PortMaster Game Packaging Guide](https://portmaster.games/packaging.html#creating-a-pull-request)

### CODE REVIEW

**Review by kloptops — APPROVED** [review]
👍

File: ports/drilbertiidiggeredoo/port.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
+{
+    "version": 2,
+    "name": "drilbertiidiggeredoo.zip",
+    "items": [
+        "Drilbert II Diggeredoo.sh",
+        "drilbertiidiggeredoo/"
```
Comment (Cebion) [inline ports/drilbertiidiggeredoo/port.json:6]: / after the portname in items

Comment (kloptops) [reply ports/drilbertiidiggeredoo/port.json:6]: That's how it is supposed to be?

File: ports/drilbertiidiggeredoo/Drilbert II Diggeredoo.sh [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+else
+  cp patch/gfx/*.png gamedata/gfx/
+  patch gamedata/main.lua < patch/main.lua.diff
+  patch gamedata/render.lua < patch/render.lua.diff
+  touch patch/patched_true
+  echo "patching complete"
+fi
+
+$GPTOKEYB "love.${DEVICE_ARCH}" -c "./drilbertiidiggeredoo.gptk" &
+./bin/love.${DEVICE_ARCH} gamedata 2>&1 | tee -a ./log.txt
```
[the remark this answers is not in the document]
Comment (tabreturn) [reply ports/drilbertiidiggeredoo/Drilbert II Diggeredoo.sh]: Thanks. Done

File: ports/drilbertiidiggeredoo/port.json (hunk 1 above)
Comment (tabreturn) [reply ports/drilbertiidiggeredoo/port.json:6]: Will wait for Cebion and Kloptops to decide who is right :)

Comment (Cebion) [reply ports/drilbertiidiggeredoo/port.json:6]: > That's how it is supposed to be?

i checked 8 ports or so and not one has / behind it :D

Comment (Cebion) [reply ports/drilbertiidiggeredoo/port.json:6]: Freshly generated from portmaster.games

```
{
  "version": 3,
  "name": "test.zip",
  "items": [
    "Test.sh",
    "test"
  ],
  "items_opt": [],
  "attr": {
    "title": "Testgame",
    "porter": [
      "Cebion"
    ],
    "desc": "asd",
    "desc_md": "ads",
    "inst": "asd",
    "inst_md": "ads",
    "genres": [
      "adventure"
    ],
    "image": null,
    "rtr": false,
    "exp": false,
    "runtime": null,
    "reqs": [],
    "arch": [
      "x86_64"
    ]
  }
}```

Comment (tabreturn) [reply ports/drilbertiidiggeredoo/port.json]: Cool -- removed the trailing slash

**Review by kloptops — APPROVED** [review]
👍

**Review by Cebion — APPROVED** [review]

**Pull request merged**